about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <script defer src="./scripts.js"></script>
+
+    <title>Avengers Conference</title>
+</head>
+
+<body>
+    <header class="mobile-header">
+        <div class="icon1">
+            <img src="images/mobil_menu.png" alt="hamburger menu" onClick="menuOpen(1)" />
+        </div>
+        <div class="logo">
+            <a href="./index.html "> <img src="./images/logo_bogota.jpg" alt="logo" /> </a>
+        </div>
+        <ul class="menu-deployed">
+            <li><a href="./index.html " onClick="menuClose(1);">Home</a></li>
+            <li><a href="#program " onClick="menuClose(1);">Program</a></li>
+            <li><a href="#speakers " onClick="menuClose(1);">Heros</a></li>
+            <li><a href="#sponsors" onClick="menuClose(1);">Sponsor</a></li>
+
+
+            <!--   <li class="mail">
+                <img src="images2/union.png" alt="email" />
+            </li> -->
+        </ul>
+
+    </header>
+
+    <div id="mobile_menu">
+        <div id="mobile_button" class="close_button" onClick="menuClose(1);">
+            <img src="images/mobil_close_menu.png" alt="close button" />
+        </div>
+        <ul>
+            <li><a href="./index.html " onClick="menuClose(1);">Home</a></li>
+            <li><a href="#program " onClick="menuClose(1);">Program</a></li>
+            <li><a href="#speakers " onClick="menuClose(1);">Heros</a></li>
+            <li><a href="#sponsors" onClick="menuClose(1);">Sponsor</a></li>
+        </ul>
+    </div>
+    <main>
+
+        <section id="cover" class="cover">
```

Comment:
- [ ] The top section should have a centered column that includes a page title, a text block with the page description, and a line with contact information or any other prompt under it, please consider customizing this section for the ```About page```, adding contact info and some information to the paragraph